.github/workflows/manual-test.yaml

```diff
@@ -0,0 +1,76 @@
+name: Manual Test
+
+on:
+  workflow_dispatch:
+    inputs:
+      conda-version:
+        description: 'Conda Version'
+        required: true
+        default: '23.1.0'
+        type: choice
+        options:
+        - 'main'
+        - '23.1.0'
```

Comment:
I wouldn't give options here, but simply document you can use versions or a branch name.